app/models/maestrano/connector/rails/concerns/entity.rb

```diff
@@ -242,7 +242,10 @@ def get_connec_entities(last_synchronization_date = nil)
     end
 
     entities.flatten!
-    Maestrano::Connector::Rails::ConnectorLogger.log('info', @organization, "Received data: Source=Connec!, Entity=#{self.class.connec_entity_name}, Data=#{entities}")
+
+    sanitized_entities = sanitizer_profile? ? Maestrano::Connector::Rails::Services::DataSanitizer.new('connec_sanitizer_profile.yml').sanitize(self.class.connec_entity_name, entities) : entities
```

Comment:
Do we need this ternary, because `.sanitize` will return the args if no profile.